dev-docker

```diff
@@ -11,7 +11,7 @@ mkdir -p "$datadir"
 dockerfile="$datadir"/../Dockerfile.dev
 (
   sed -e '1,/WORKDIR/!d' "$basedir"/Dockerfile
-  echo 'RUN apt-get install -q2 snakefood flake8 git python3-wheel python3-django-extensions ipython3 rabbitmq-server chromium nodejs ruby-foreman procps && \'
+  echo 'RUN apt-get install -q2 snakefood flake8 git python3-wheel python3-django-extensions python3-djangorestframework-extensions ipython3 rabbitmq-server chromium nodejs ruby-foreman procps && \'
```

Comment:
Dependencies here are just when running tests in docker. What you added should not be a dev thing, which you did correctly above adding to Dockerfile. Adding here is redundant